samples/app-sso/csharp/App SSO Sample/Bots/DialogBot.cs

```diff
@@ -38,8 +41,10 @@ public DialogBot(ConversationState conversationState, UserState userState, T dia
         {
             _connectionName = configuration["ConnectionName"] ?? throw new NullReferenceException("ConnectionName");
             _userState = userState ?? throw new NullReferenceException(nameof(userState));
+            _conversationState = conversationState ?? throw new NullReferenceException(nameof(conversationState));
             _logger = logger;
-            _dialogManager = new DialogManager(dialog) { ConversationState = conversationState, UserState = userState };
+            _dialog = dialog;
+            // _dialogManager = new DialogManager(dialog) { ConversationState = conversationState, UserState = userState };
```

Comment:
Are we not using dialog manager for this sample ?

Reply:
Updated to directly use dialog methods. It is also depticated